conanfile.py

```diff
@@ -133,7 +133,9 @@ def build(self):
             "BUILD_DOC"                : self.options.build_doc,
 
             # Paramaters
-            "MKL_ROOT"                 : f"/opt/intel/inteloneapi/mkl/{self.oneapi_version}",
+            # Conan does not officially support the DPC++ compiler, hence disable the compiler_id check
+            "CONAN_DISABLE_CHECK_COMPILER" : True,
+            "MKL_ROOT"                 : f"/opt/intel/inteloneapi/mkl/latest",
```

Comment:
I see the same changes for MKL_ROOT in PR #29. Do you plan to close it and extend this one with other changes or keep them separated?

Reply:
I will keep the PRs separate. Reverted that change in this PR.